build/Linux+BSD/mscore.appdata.xml.in

```diff
@@ -0,0 +1,316 @@
+  <name>MuseScore</name>
+  <summary>Create, playback, and print sheet music for free</summary>
+  <icon type="remote" height="128" width="128">https://raw.githubusercontent.com/musescore/MuseScore/master/assets/musescore-icon-round-128.png</icon>
+  <icon type="remote" height="64" width="64">https://raw.githubusercontent.com/musescore/MuseScore/master/assets/musescore-icon-round-64.png</icon>
+
+  <description>
+    <p>
+      Create, playback, and print sheet music for free. MuseScore is cross-platform, multi-lingual, open source music notation software. It features an easy to use WYSIWYG editor with audio score playback for results that look and sound beautiful. It supports unlimited staves with up to four voices each, dynamics, articulations, lyrics, chords, lead sheet notation, import/export of MIDI and MusicXML, export to PDF and WAV, plus online score sharing.
+    </p>
+    <p>
+      MuseScore can upload scores directly to the score sharing site musescore.com. Program support is provided on musescore.org.
+    </p>
+    <p>
+      Features supported by MuseScore include:
+    </p>
+    <ul>
+      <li>Unlimited score length</li>
+      <li>Unlimited number of staves per system</li>
+      <li>Up to four independent voices per staff</li>
+      <li>Score creation wizard and templates</li>
+      <li>Automatic part extraction and transposition</li>
+      <li>Repeats, including segnos, codas, and measure repeats</li>
+      <li>Dynamics, articulations, and other expression markings, with playback support for most</li>
+      <li>Custom text markings</li>
+      <li>Lyrics</li>
+      <li>Chord symbols</li>
+      <li>Jazz notation, including lead sheets, slash notation and a “handwritten” font for text</li>
+      <li>Swing and shuffle playback</li>
+      <li>Mixer for instrument levels and effects</li>
+      <li>Percussion notation</li>
+      <li>Early music notation</li>
+      <li>Cross-staff beaming</li>
+      <li>Import of graphics</li>
+      <li>Custom key signatures</li>
+      <li>Additive time signatures</li>
+      <li>User-defined score styles</li>
+    </ul>
+    <p>
+      Most elements in MuseScore are laid out automatically but can also be positioned manually. The capabilities of MuseScore can be extended via plugins, and the growing repository on musescore.org contains many plugins submitted by users.
+    </p>
+    <p>
+      MuseScore includes a set of sounds that reproduce common instruments (as defined by General MIDI) without taking up a lot of disk space or memory, but you can also substitute any SoundFont you prefer for a wider variety of sounds or for more realism.
+    </p>
+    <p>
+      MuseScore can import and export MIDI and MusicXML files, and it can also import from Capella and several other programs. MuseScore can export to PDF, PNG, and other graphic formats, to WAV and other audio formats, or to Lilypond for an alternative layout and print option.
+    </p>
+  </description>
+
+  <categories>
```

Comment:
I would add SheetMusic, NotationSoftware categories.

Reply:
I just copied them over from the desktop file (now that I think of it I need to check whether I _should_ have done that, I have a feeling that should be done automatically).

Reply:
Yeah, it’s pulled from the desktop file, so this is a different issue.